src/dist.rs

```diff
@@ -0,0 +1,321 @@
+use crate::{
+    anyhow::{ensure, Context, Result},
+    camino, clap, default_build_command, metadata,
+};
+use lazy_static::lazy_static;
+use std::{fs, path::PathBuf, process};
+use wasm_bindgen_cli_support::Bindgen;
+
+/// A helper to generate the distributed package
+///
+/// This struct provide a customizable way to assemble and generate a package
+/// for Wasm.
```

Comment:
The second paragraph is repetition of the first.
```suggestion
/// A helper to generate the distributed package.
```